etc/install-zstd.sh

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
```suggestion
#!/bin/sh
```

(or `bash` if that's what you're assuming here)